tests/core/simulation_control_05.cc

```diff
@@ -0,0 +1,177 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2020 - by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 3.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+
+*
+* Author: Bruno Blais, Polytechnique Montreal, 2020-
+*/
+
+/**
+ * @brief This test checks that the time-step round-off errors
+ * do not affect the end of the simulation
+ */
```

Comment:
DId you modify an existing or create a new one? Change the header and add an explanation of what the test does.